help/actions/_posts/2015-02-09-export.md

```diff
@@ -54,6 +54,13 @@ Accomazzi2019AAS            -- %1H%Y%q
 AccomazziKurtz2019          -- %2H%Y
 ```
 
+Note there is also `%zm` specifier that is used for enumeration. This format outputs sequence of characters to the end of `keyformat` when it is not unique among selected records. For example having `keyformat` of `%1H%Y%zm` produces:
```

Comment:
- small typo: "This format outputs **a** sequence of characters..."
- also how about clarifying which characters are added, since it's non-configurable? Like this: "This format outputs a sequence of characters (i.e. a, b, c, ...) to the end..."

Reply:
thank you.